packages/firebase-functions/push/functions/src/routes/push.ts

```diff
@@ -29,7 +29,7 @@ export const push = async (req: Request): Promise<IResponse> => {
       code: 200,
       success: true,
     };
-  } catch (error) {
+  } catch (error: any) {
```

Comment:
Would it be better to start with `Error` as the default type for `catch`, so that any non-standard error properties require explicit checking?

Reply:
I thought so too, but Typescript didn't agree with us: https://fettblog.eu/typescript-typing-catch-clauses/

Reply:
you could use an `unknown` instead which means that in your catch you need to do positive work to understand what the type of the error is.